R/reporter.R

```diff
@@ -34,7 +38,8 @@ Reporter <- R6::R6Class("Reporter",
         warning("append ignored", call. = FALSE)
       }
 
-      cat(..., file = self$out, sep = sep, fill = fill, labels = labels)
+      cat(..., file = self$out, sep = sep, fill = fill, labels = labels,
```

Comment:
Can you please use tidyverse style here? http://style.tidyverse.org/syntax.html#long-lines